### PR TITLE
Kevlar wetsuit tweaks

### PR DESCRIPTION
**Pull request by anoobindisguise**
#### Summary
Balance "updates to Kevlar wetsuit"

#### Purpose of change
Kevlar wetsuits weren't SKINTIGHT despite being made of a skintight wetsuit. Also, there was no variant for the thick wetsuit.

#### Describe the solution
Splits kevlar wetsuit into two options: regular and thick, made with the respective wetsuits. They occupy SKINTIGHT, NORMAL layers both instead of just the NORMAL layer. Environmental protection lowered a bit. Warmth and encumbrance increased (regular kevlar wetsuit had less encumbrance than its constituent wetsuit). Crafting made slightly more complicated, as you now need silicone caulk to act as a sealant.

### CODE REVIEW

**Comment by nopenoperson** [comment]
silicone caulk wouldnt stick to neoprene foam. super glue would work a lot better. blindstiching with a curved needle with superglue and tape to fix any horrible accidents is way more realistic

**Comment by nopenoperson** [comment]
rubber cement would also work

**Comment by anoobindisguise** [comment]
> You posted this in ready to merge but it's still in draft.

Sorry! I forgot about that when editing the material.

File: data/json/recipes/armor/bespoke_armor/survivor.json [hunk 1]
```diff
@@ -49,35 +49,69 @@
     "difficulty": 7,
     "time": "12 h",
     "autolearn": true,
-    "using": [ [ "sewing_kevlar", 200 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
+    "using": [ [ "sewing_kevlar", 200 ], [ "tailoring_neoprene_patchwork", 30 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/bespoke_armor/survivor.json:52]: You have removed kevlar from the list of the components, so resulting item should not be called "Kevlar" anymore or components list should be fixed (same for recipes below).

**Pull request merged**